README.md

```diff
@@ -12,6 +12,6 @@ The project introduces the Argo Rollout dashboard into the Argo CD Web UI.
 
 ```
 kubectl apply -n argocd \
-    -f https://raw.githubusercontent.com/argoproj-labs/rollout-extension/v0.1.0/manifests/install.yaml
+    -f https://raw.githubusercontent.com/argoproj-labs/rollout-extension/master/manifests/install.yaml
```

Comment:
We should keep this as a fixed version (instead of `master`) and instead update the README on each release with the new version